version/check.go

```diff
@@ -0,0 +1,64 @@
+package version
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/hashicorp/go-version"
+	"github.com/pkg/errors"
+)
+
+const versionTimeout = time.Second * 1
+
+// CheckLatestVersion prints the latest version if the current version is less than latest version
```

Comment:
I think it should return the result instead of printing the version. Let the application handle the formatting etc.